sdk/eventhub/azure-eventhub/azure/eventhub/_client_base.py

```diff
@@ -173,6 +174,24 @@ def get_token(self, *scopes, **kwargs):  # pylint:disable=unused-argument
             raise ValueError("No token scope provided.")
         return _generate_sas_token(scopes[0], self.policy, self.key)
 
+class AzureNamedKeyTokenCredential(object):
```

Comment:
Anna mentioned in my [ServiceBus GA Azure SAS Credential PR](https://github.com/Azure/azure-sdk-for-python/pull/18264#discussion_r623216248) that
> Let's give this a more specific name to avoid any developer confusion between this and the Core credential object.
Something like ServiceBusSasToken

So I think we'd better rename this class to something like `EventhubAzureNamedKeyCredential`?

Reply:
renamed both